index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2><a href="#">Welcome</a></h2>
+        <i class="fa fa-bars" aria-hidden="true"></i>
+    </header>
+    <section class="home-page">
+        <h1 class="intro">Hey there. I’m Tobin <br><span>I’m a software<br> developer</span></h1>
+
+        <p class="info"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p>
+            <ul class="icon-list">
+                <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i>
+                </a></li>
+                <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i>
+                </a></li>
+                <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i>
+                </a></li>
+                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i>
+                </a></li>
+                <li><a href="#"><i class="fa fa-angellist" aria-hidden="true"></i>
+                </a></li>
+            </ul>
```

Comment:
- [ ] OPTIONAL Please try to format your code correctly. it makes it easier for other developers to read your code.